src/server.cpp

```diff
@@ -567,10 +569,10 @@ void draconity_mimic_done(int key, dsx_mimic *mimic) {
         draconity->mimic_lock.unlock();
         uint64_t client_id = mimic_info.first;
         uint32_t tid = mimic_info.second;
-        draconity_publish_one("mimic",
-                              BCON_NEW("cmd", "mimic.done",
-                                       "tid", BCON_INT32(tid)),
-                              client_id);
+        draconity_send("mimic",
+                       success_msg(),
```

Comment:
Before there were two mimic messages, one for scheduled, one for completion. This should be the only mimic-related message now.